_layouts/partners/impact-story.html

```diff
@@ -18,6 +18,13 @@ <h1 class="partners-title">{{ page.title }}</h1>
                     <div class="partners-body padding-top-2">{{ page.challenge | markdownify }}</div>
                     <div class="partners-body padding-top-2">{{ page.solution | markdownify }}</div>
                     <div class="partners-body padding-top-2">{{ page.impact | markdownify }}</div>
+                    {% if page.callout_tips %}                    
+                        <div class="grid-row grid-gap-6">
```

Comment:
I see this is consistent with the existing indentation of the file 👍 , though I think the existing indentation is non-standard with the typical 2-space indentation we use elsewhere. Maybe a follow-up refactor to consider.